app/experimenter/nimbus-ui/src/hooks/useConfig.tsx

```diff
@@ -0,0 +1,31 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { useApolloClient } from "@apollo/client";
+import { GET_CONFIG_QUERY } from "../gql/config";
+import { getConfig } from "../types/getConfig";
+import serverConvig from "../services/config";
+
+/**
+ * Hook to retrieve GraphQL and Server config valyes.
+ *
+ * NOTE: This hook is dependent on the initial query to retrieve
+ * GraphQL config being performed inside the <App> component. Do
+ * not use before this component is rendered.
+ *
+ * Example:
+ *
+ * const { channels, probeSets } = useConfig();
+ */
+
+export function useConfig() {
```

Comment:
Maybe a non-thing, but: Could we rename this to something like `useServerConfig` or `useGQLConfig`? We already have [config from the meta header](https://github.com/mozilla/experimenter/blob/main/app/experimenter/nimbus-ui/src/services/config.ts), so there's a bit of a name collision there.